loadgenerator/loadgen

```diff
@@ -458,6 +496,13 @@ def _get_parser():
             description='Show the status of all load generator jobs whether running or not')
     subparser.set_defaults(func=status)
 
+    subparser = subparsers.add_parser('autorun',
```

Comment:
I got confused by the difference between `autostart` and `autorun`. Is `autostart` going away? Or are they two different things?

Reply:
autostart is the original one using GCE, it's what's called in the install script - wasn't sure if I wanted to get rid of it yet